WebViewControl/AsyncResourceHandler.cs

```diff
@@ -9,6 +9,7 @@ internal class AsyncResourceHandler : DefaultResourceHandler {
 
         private CefCallback responseCallback;
         private bool autoDisposeStream;
+        private object SyncRoot = new object();
```

Comment:
to maintain the consistency used in this project:

```
private bool autoDisposeStream;

private object SyncRoot { get; } = new object();

Reply:
done ☑️